packages/next/src/server/normalizers/built/app/app-page-normalizer.ts

```diff
@@ -1,12 +1,35 @@
 import { PAGE_TYPES } from '../../../../lib/page-types'
 import { AbsoluteFilenameNormalizer } from '../../absolute-filename-normalizer'
+import { Normalizers } from '../../normalizers'
 
 /**
  * DevAppPageNormalizer is a normalizer that is used to normalize a pathname
  * to a page in the `app` directory.
  */
-export class DevAppPageNormalizer extends AbsoluteFilenameNormalizer {
+class DevAppPageNormalizerInternal extends AbsoluteFilenameNormalizer {
   constructor(appDir: string, extensions: ReadonlyArray<string>) {
     super(appDir, extensions, PAGE_TYPES.APP)
   }
 }
+
+export class DevAppPageNormalizer extends Normalizers {
+  constructor(
+    appDir: string,
+    extensions: ReadonlyArray<string>,
+    _isTurbopack: boolean
```

Comment:
nit: is this is going to be permanent later, would be nice to avoid the boolean argument